### PR TITLE
First phase of docs refacts. Started to document script usage

### PR DESCRIPTION
**Pull request by MatanHeledPort**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Added docs pages

Please also include the path for the added docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/kubernetes/full-kubernetes-exporter.md [hunk 1]
```diff
@@ -35,26 +68,26 @@ In the Git repository, you can find 10 `.tf` files which will create the followi
...
-`Service` uses selectors to route traffic to pods. Since this is not a direct mapping, relating a service to a specific pod is only possible either by activley mapping pods to their service, or using a strict naming convention. In this use case, you we will only map the service's selectors.
-:::
+* Deployment
+* ReplicaSet
+* StatefulSet
+* DaemonSet
+* Job
+
+- `Service` uses selectors to route traffic to pods. Since this is not a direct mapping, relating a service to a workload is usually done using a strict naming convention. This use-case assumes a naming convention where the `Workload`'s name is equal to its associated `service`.
+  :::
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/kubernetes/full-kubernetes-exporter.md:90]: You have some formatting issue with the admonition block, fix it

**Review by MPTG94 — APPROVED** [review]
LGTM just make sure to do another pass to see that the content matches the current away of deploying, the script vars, blueprints json vs tf etc

**Comment by MatanHeledPort** [comment]
This will be merged after the new script is merged

File: docs/build-your-software-catalog/sync-data-to-catalog/kubernetes/full-kubernetes-exporter.md [hunk 2]
```diff
@@ -19,6 +18,40 @@ Using Port's Kubernetes Exporter, you can keep track of your K8s resources and e
...
+`CLUSTER_NAME` - The cluster's name as it will be exported to Port (Default: `my-cluster`)
+:::tip
+The script replaces the string `{CLUSTER_NAME}` from the config yaml and with the value of the environment variable `CLUSTER_NAME`. This is useful for when creating a generic `config.yaml`.
+:::
+
+#### Helm Chart Installation configuration
+
+`CONFIG_YAML_URL` - supports raw text file URLs in https format `https://domain.com/file/path/config.yaml` and also local paths `clusters/production/config.yaml` (Default: `https://github.com/port-labs/template-assets/blob/main/kubernetes/kubernetes_config.yaml`)
+
+``
```
[the remark this answers is not in the document]
Comment (MatanHeledPort) [reply docs/build-your-software-catalog/sync-data-to-catalog/kubernetes/full-kubernetes-exporter.md:53]: Why?

**Pull request merged**